frame/system/src/lib.rs

```diff
@@ -879,12 +898,118 @@ impl<T: Trait> Module<T> {
 		let xts_root = extrinsics_data_root::<T::Hashing>(extrinsics);
 		<ExtrinsicsRoot<T>>::put(xts_root);
 	}
+
+	/// An account is being created.
+	pub fn on_created_account(who: T::AccountId) {
+		T::OnNewAccount::on_new_account(&who);
+		Self::deposit_event(RawEvent::NewAccount(who));
+	}
+
+	/// Kill the account and reap any related information.
+	pub fn kill_account(who: T::AccountId) {
+		if Account::<T>::contains_key(&who) {
+			Account::<T>::remove(&who);
+			Self::on_killed_account(who);
+		}
+	}
+
+	/// Do anything that needs to be done after an account has been killed.
+	fn on_killed_account(who: T::AccountId) {
+		T::OnReapAccount::on_reap_account(&who);
```

Comment:
```suggestion
		T::OnKilledAccount::on_reap_account(&who);
```

Probably best to update the name across the board here.